pkg/serverless/trace/propagation/extractor_test.go

```diff
@@ -407,6 +407,48 @@ func TestExtractorExtract(t *testing.T) {
 			expNoErr: true,
 		},
 
+		// events.EventBridgeEvent
+		{
+			name: "eventbridge-event-empty",
+			events: []interface{}{
+				events.EventBridgeEvent{},
+			},
+			expCtx:   nil,
+			expNoErr: false,
+		},
+		{
+			name: "eventbridge-event-with-trace-context",
+			events: []interface{}{
+				events.EventBridgeEvent{
+					Detail: struct {
+						TraceContext map[string]string `json:"_datadog"`
+					}{
+						TraceContext: map[string]string{
+							"x-datadog-trace-id":          "123456789",
+							"x-datadog-parent-id":         "987654321",
+							"x-datadog-sampling-priority": "1",
```

Comment:
Use the `headersDD` instead. You should also test the other header types `headersAll` and `headersW3C`. Take a look at the tests above and follow that pattern.